<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QWidget" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1539</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>611</height>
    </rect>
   </property>
   <widget class="QPushButton" name="ExitPushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Page3pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>扩展应用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Page2pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>审计管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Page1pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>代码审计</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="ChangeStackedWidget">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>9</y>
     <width>1471</width>
     <height>841</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page1"/>
   <widget class="QWidget" name="page2"/>
   <widget class="QWidget" name="page3"/>
  </widget>
  <widget class="QPushButton" name="HidePushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>820</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
